www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt

```diff
@@ -111,12 +111,12 @@
                     // Save the form before continuing
                     saveFormToEndpoint("/api/caddy/general/set", 'frm_GeneralSettings',
                         function() {  // callback_ok: What to do when save is successful
-                            showAlert("Configuration saved successfully. Please don't forget to apply the configuration.", "Save Successful");
+                            showAlert("{{ lang._('Configuration saved successfully. Please do not forget to apply the configuration.') }}", "{{ lang._('Save Success') }}");
```

Comment:
ah the old contractions-are-not-for-technical-writing is enforced here due to the weird quoting rules of volt strings ;)

Reply:
I found out the hard way. I will avoid them.